imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePickerActivity.java

```diff
@@ -233,6 +234,19 @@ private void invalidateTitle() {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         getMenuInflater().inflate(R.menu.ef_image_picker_menu_main, menu);
+        for (int i = 0; i < menu.size(); i++) {
```

Comment:
This will change all toolbars options menu color right?

Reply:
This will loop thought every menu then get icon and change color at line 243